src/pybind/mgr/mgr_module.py

```diff
@@ -309,6 +311,8 @@ def _parse_args(self):
 
     def __call__(self, func):
         self.func = func
+        if not self.desc:
+            self.desc = inspect.getdoc(func)
```

Comment:
Nice improvement! We are close to infer the command name from the function (__ => whitespace, _ => -) and the argument types from type annotations:
```py
@CLICommandRead
def device__predict_life_expectancy(devid: CephString):
...
```